data/src/main/java/org/akvo/flow/data/entity/SyncedTimeMapper.java

```diff
@@ -18,11 +18,29 @@
  *
  */
 
-package org.akvo.flow.data.util;
+package org.akvo.flow.data.entity;
 
-public class Constants {
+import android.database.Cursor;
 
-    public static final int IMAGE_SIZE_320_240 = 0;
-    public static final int IMAGE_SIZE_640_480 = 1;
-    public static final int IMAGE_SIZE_1280_960 = 2;
+import org.akvo.flow.database.SyncTimeColumns;
```

Comment:
Did you rename this class to something else or is this just the way it was interpreted by git?